ztp/policygenerator-kustomize-plugin/kustomization.yaml

```diff
@@ -4,3 +4,18 @@ generators:
 - testPolicyGenTemplate/group-du-sno-ranGen.yaml
 - testPolicyGenTemplate/site-du-sno-1-ranGen.yaml
 - testPolicyGenTemplate/site-du-sno-2-ranGen.yaml
+- testPolicyGenTemplate/group-du-sno-validator-ranGen.yaml
+
+patches:
+- patch: |-
+    apiVersion: policy.open-cluster-management.io/v1
+    kind: Policy
+    metadata:
+      annotations:
+        # first time TALM observes this policy as compliant, it won't move on.
+        # In future reconcilations if the specified number of seconds has passed 
+        # since the first compliancy time and policy is still compliant, then TALM
+        # will move on from this policy
+        ran.openshift.io/soak-seconds: "30"
+      name: group-du-sno-validator-du-policy
+      namespace: ztp-group
```

Comment:
We need a comment that the name/namespace here **must match** the values in the validator policy.